CRITICAL - 2024-01-22 19:36:43 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\mm-management\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\Exceptions.php(149): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Except
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Page Not Found
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(989): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound(NULL)
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(370): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\mm-management\public\index.php(79): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#4 {main}
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-01-22 19:39:41 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\mm-management\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\Exceptions.php(149): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Except
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Page Not Found
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(989): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound(NULL)
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(370): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\mm-management\public\index.php(79): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#4 {main}
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-01-22 19:39:43 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\mm-management\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Debug\Exceptions.php(149): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Except
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Page Not Found
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(989): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound(NULL)
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(370): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\mm-management\public\index.php(79): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#4 {main}
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-01-22 20:53:27 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'mm-management', 3306, '', 0)
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\mm-management\app\Controllers\fr_index.php(58): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(942): App\Controllers\fr_index->gallery()
#8 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\fr_index))
#9 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mm-management\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\mm-management\app\Controllers\fr_index.php(58): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(942): App\Controllers\fr_index->gallery()
#7 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\fr_index))
#8 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mm-management\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mm-management\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-01-22 20:53:27 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `mm_gl_ket`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\fr_index.php(58): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\fr_index->gallery()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\fr_index))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mm-management\\public\\index.php')
